src/ui/public/chrome/directives/app_switcher/app_switcher.js

```diff
@@ -1,10 +1,18 @@
 import DomLocationProvider from 'ui/dom_location';
 import { parse } from 'url';
-import { bindKey } from 'lodash';
+import { bindKey, filter } from 'lodash';
 import '../app_switcher/app_switcher.less';
 import uiModules from 'ui/modules';
 import appSwitcherTemplate from './app_switcher.html';
 
+function isNavLinkEnabled(link) {
+  return link.enable === undefined || link.enable;
+}
+
+function isNavLinkShown(link) {
+  return link.show === undefined || link.show;
```

Comment:
Because the `show` property may not be set (default case). In that case we want to show the link.